lib/support/cli.js

```diff
@@ -254,6 +255,11 @@ module.exports.parseCommandLine = function parseCommandLine() {
     }
     // End hack
 
+    // FIXME: Find a solution that would work for all aliasing
```

Comment:
Thinking maybe we could find a solution for when we parse which plugins that will be used or @tobli do you have a nice solution lined up for it?